packages/salesforcedx-vscode-core/package.json

```
@@ -837,6 +837,11 @@
       "type": "object",
       "title": "%feature_previews_title%",
       "properties": {
+        "salesforcedx-vscode-core.force-push-on-save": {
```

```suggestion
        "salesforcedx-vscode-core.push-or-deploy-on-save.overrideConflictsOnPush": {
```

Can we move this setting right below the `push-or-deploy-on-save.enabled` entry to group related setting?

@smaddox-sf @sbudhirajadoc what do you think of my suggested setting name (Override conflicts on push)? Also moved the setting into the `push-or-deploy-on-save` namespace.

---

done

---

Looks good on both.  @sbudhirajadoc will do a final review on the setting name & description

---

added a small update on description. setting name is good.